azkaban-common/src/test/java/azkaban/user/XmlUserManagerTest.java

```diff
@@ -159,12 +162,78 @@ public void testAutoReload() throws Exception {
         System.out.println("Test failed " + e.getMessage());
         fail("Test failed " + e.getMessage());
       }
-    } catch (ConditionTimeoutException te) {
-      System.out.println("The config did not reload in 60 seconds");
-      fail("The config did not reload in 60 seconds");
+    } catch (final ConditionTimeoutException te) {
+      System.out.println("The config did not reload in 30 seconds");
```

Comment:
println seems redundant, `fail` message below should be enoguh?

Reply:
I added them for debugger as fail message does not show up on it's console. I will remove them.
Thanks